### ISSUE TITLE
`Edit Alias' improperly displaying typed `Content' value 

### ISSUE THREAD

**Issue by pablo-blueoakdb** [opnsense/core#4066]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[-] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md

[-] I have searched the existing issues and I'm convinced that mine is new.

**Describe the bug**
The **Content** column in the **Edit Alias** screen only displays the last, vertical half of the character typed in.

*Tip*: to validate your setup was working with the previous version, use opnsense-revert (https://docs.opnsense.org/manual/opnsense_tools.html#opnsense-revert)

**To Reproduce**
1. Go to **Firewall > Aliases**
2. Click on '+' to create a new alias
3. Set the **Type** to **Host(s)**
4. See the error as you attempt to enter 1.2.3.4 in the **Content** field.

**Expected behavior**
The entire string typed in should be displayed left to right as it is being typed.

**Environment**
OPNsense 20.1.5 (amd64/OpenSSL)

**Comment by AdSchellevis** [comment]
browser issue?
![image](https://user-images.githubusercontent.com/9783985/80289729-d20dd500-8740-11ea-9406-b05f05638e0b.png)
![image](https://user-images.githubusercontent.com/9783985/80289733-da661000-8740-11ea-83ba-48f03221fb6a.png)

**Comment by pablo-blueoakdb** [comment]
Interesting!  I'm using Brave:  Version 1.7.98 Chromium: 81.0.4044.113 (Official Build) (64-bit) - I'm on openSUSE.

This is a screenshot as I enter input:

![b4](https://user-images.githubusercontent.com/2430861/80289819-4760b200-870f-11ea-936d-6e1ca9774d03.png)

This is a screenshot after I've entered it (you can see the values I entered):

![aft](https://user-images.githubusercontent.com/2430861/80289834-5e9f9f80-870f-11ea-9c02-0f11e9f6b669.png)

**Comment by pablo-blueoakdb** [comment]
Hi again,

I gathered some more data.

**TL;DR**
I see the problem even with a new profile.

**Details**
I created a new Linux user and ran Brave.  In this invocation no extensions or anything.  Plain vanilla Brave.  

Unfortunately the problem is still there.

**Comment by pablo-blueoakdb** [comment]
Hmm, as I'm new to OPNsense, I tore down and rebuilt my set up.  The problem is gone.  Hmmm.
